layouts/membership/section.html

```diff
@@ -20,8 +20,4 @@ <h2 id="member-testimonials">Member Testimonials</h2>
 	</div>
   </div>
 </main>
-{{ end }}
-
-{{ define "footer-suffix" }}
-  <script src="/js/index.js"></script>
```

Comment:
This keeps causing error not found, the index.js file does not exist, and it stops my js for dropdowns.